bin/pyenv-installer

```diff
@@ -72,14 +72,16 @@ if ! command -v pyenv 1>/dev/null; then
   { echo "# Load pyenv automatically by adding"
     echo "# the following to ${profile}:"
     echo
+    pyenv_bin=${PYENV_ROOT-$HOME}/bin
```

Comment:
`.pyenv` directory has been vanished from `pyenv_bin`. This 2 lines should be replaced by `pyenv_bin="${PYENV_ROOT:-\${HOME\}/.pyenv}/bin"`.